<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>550</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton, #pushButton_6, #pushButton_7{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}

QPushButton#pushButton:hover, #pushButton_6:hover, #pushButton_7:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed, #pushButton_6:pressed, #pushButton_7:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}

QPushButton#pushButton_2, #pushButton_3, #pushButton_4, #pushButton_5{
	background-color: rgba(0, 0, 0, 0);
	color:rgba(85, 98, 112, 255);
}

QPushButton#pushButton_2:hover, #pushButton_3:hover, #pushButton_4:hover, #pushButton_5:hover{
	color: rgba(131, 96, 53, 255);
}

QPushButton#pushButton_2:pressed, #pushButton_3:pressed, #pushButton_4:pressed, #pushButton_5:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91, 88, 53, 255);
}

</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>280</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/images/image3.png);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>280</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 80);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>240</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border-bottom-right-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 200);</string>
    </property>
    <property name="text">
     <string>Diagnostic du 
Cancer du Sein</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>220</width>
      <height>315</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0, 0, 0, 200);</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>110</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>  Email Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>85</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>  First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>85</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>  Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>160</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>  Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>210</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>  Confirm Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
